snapcraft.yaml

```diff
@@ -1653,3 +1654,4 @@ parts:
       wrappers/remote-viewer: bin/
       wrappers/sshfs: bin/
       wrappers/virt-v2v-in-place: bin/
+      wrappers/journalctl: bin/
```

Comment:
@tomponline you suggested not using `run-host` for `journalctl` interception but didn't elaborate as to why you preferred a separated wrapper.

I think that having `daemon.start` do:

```
ln -s "${SNAP}/wrappers/run-host" "/run/bin/journalctl"
```

Would achieve the same and "shadow" `journalctl` only for LXD's snap. So same as what this separated wrapper would do but without duplicating it.